core/services/chainlink/testdata/secrets-full.toml

```diff
@@ -6,3 +6,5 @@ ExplorerSecret = "secret"
 
 KeystorePassword = "keystore_pass"
 VRFPassword = "VRF_pass"
+
+PyroscopeAuthToken = "pyroscope-token"
```

Comment:
@samsondav @bolekk I am thinking about aligning the secrets format with the regular config, since some of the same groupings exist in both:
```toml
[Database]
URL = "postgresql://user:pass@localhost:5432/dbname?sslmode=disable"
BackupURL = "postgresql://user:pass@localhost:5432/backupdbname?sslmode=disable"

[Explorer]
AccessKey = "access_key"
Secret = "secret"

[Password]
Keystore = "keystore_pass"
VRF = "VRF_pass"

[Pyroscope]
AuthToken = "pyroscope-token"
```
Which can also be expressed as dot separated:
```toml
Database.URL = "postgresql://user:pass@localhost:5432/dbname?sslmode=disable"
Database.BackupURL = "postgresql://user:pass@localhost:5432/backupdbname?sslmode=disable"
Explorer.AccessKey = "access_key"
Explorer.Secret = "secret"

Password.Keystore = "keystore_pass"
Password.VRF = "VRF_pass"

Pyroscope.AuthToken = "pyroscope-token"
```
WDYT?

Reply:
Done